### ISSUE TITLE
Captchas are not being solved

### ISSUE THREAD

**Issue by Exonip** [iv-org/invidious#2031]
My ip got blocked and I can acess youtube only after solving a captcha using the server's ip.
I added a captcha key in the config but captchas are not being solved.
I don't even get the block message, only the following error:
`[error] UpdateDecryptFunctionJob : Nil assertion failed`
In the frontend it says the instance is blocked though

**Comment by Exonip** [comment]
Probably an instance-issue, so please adjust the labels

**Comment by TheFrenchGhosty** [comment]
@unixfox I think you had this issue before?

**Comment by Exonip** [comment]
Yes, added funds and the key is 100% correct. I looked at the anti-captcha dashboard and the server's IP is not listed so there seem to be no requests at all

**Comment by unixfox** [comment]
Maybe you are partially blocked. Is every page returning an error about being blocked?

**Comment by Exonip** [comment]
I just checked again and the instance doesn't seem to be blocked anymore. Before it only said the instance would be likely blocked below the video but it played just fine. There still is not a single request to anti-captcha.com though

**Comment by unixfox** [comment]
I used to get the same exact issue, but the issue disappeared after a few weeks.

**Comment by Exonip** [comment]
Do you think it has anything to do with the error `[error] UpdateDecryptFunctionJob : Nil assertion failed`?
I set decrypt_polling: false in the config because it was mentioned in another issue and it seems to have fixed the errors

**Comment by Exonip** [comment]
I also found the error `2021-04-21 19:51:52 UTC [error] BypassCaptchaJob: Read timed out` in the console now

**Comment by SamantazFox** [comment]
> Do you think it has anything to do with the error `[error] UpdateDecryptFunctionJob : Nil assertion failed`?
> I set decrypt_polling: false in the config because it was mentioned in another issue and it seems to have fixed the errors

This error is because the background job that should download and parse the player's javacript to extract some DRM decryption function is broken.
